README.md

```diff
@@ -1,5 +1,3 @@
-# github-assignment
```

Comment:
I think you deleted this line by accident. Please could you add it back in a subsequent commit? thanks

Reply:
Dr. West, where am I supposed to add that?

Reply:
Alright! Done!